docs/README.rst

```diff
@@ -1,27 +1,47 @@
-==============================================
 How to build AWS ParallelCluster documentation
-==============================================
+##############################################
 
-First, install the libraries needed to build Sphinx doc:
+AWS ParallelCluster documentation can be constructed using Sphinx (http://www.sphinx-doc.org/en/master/) or
+tox (https://tox.readthedocs.io/en/latest/index.html).
+
+To begin, install the required libraries:
 
 .. code-block:: sh
 
     $ pip install -r docs/requirements.txt
 
-Next, execute the :code:`make html` command.
+Select your preferred tool of choice and build the documentation using the guidelines outlined below:
+
+Sphinx
+======
+
+Sphinx was originally created to facilitate Python documentation creation.  It has excellent facilities for
+documentation of software projects for a range of languages.
+
+To build the AWS ParallelCluster documentation using Sphinx, run the :code:`make html` command from the
+the directory containing the AWS ParallelCluster source code documentation:
 
 .. code-block:: sh
 
+    $ cd ~/src/aws-parallelcluster/docs
```

Comment:
I think we should use the relavite path `$ cd docs` and not a specific path.

Reply:
Agreed and done.